### ISSUE TITLE
WPML Support?

### ISSUE THREAD

**Issue by mkormendy** [norcross/gravity-tooltips#46]
Is there any way that you can add WPML support?

**Comment by norcross** [comment]
not totally sure how i'd even do that, since it's a specific plugin

**Comment by mkormendy** [comment]
it's a pretty major specific plugin

**Comment by norcross** [comment]
yeah, I'm aware of it. but i don't use it and haven't looked into how to implement it. i'll welcome a PR if you want to work on it.
